api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -94,4 +94,9 @@ message QuicProtocolOptions {
   // If not specified, no cmsg will be saved to QuicReceivedPacket.
   repeated core.v3.SocketCmsgHeaders save_cmsg_config = 12
       [(validate.rules).repeated = {max_items: 1}];
+
+  // If true, the listener will reject connection-establishing packets at the
+  // QUIC layer by replying with an empty version negotiation packet to the
+  // client.
+  google.protobuf.BoolValue reject_new_connections = 13;
```

Comment:
In this particular case, are there any situations in the future in which we'd want a data plane to default this to true?  I suspect not, in which case we can probably just use `bool` instead of `google.protobuf.BoolValue`.